### PR TITLE
Send user agent string with each swap

### PR DESCRIPTION
**Pull request by guggero**
This PR adds the daemon's user agent string to each new swap.

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -29,8 +29,18 @@ const (
...
+	// defaultAgentName is the default name of the software that is added as
+	// the first part of the user agent string.
+	defaultAgentName = "loopd"
 )
 
+// AgentName stores the name of the software that is added as the first part of
+// the user agent string. This defaults to the value "loopd" when being run as
+// a standalone component but can be overwritten by LiT for example when loopd
+// is integrated into the UI.
+var AgentName = defaultAgentName
```
Comment (bhandras) [inline version.go:42]: When you say "can be overwritten..." how do you mean exactly? Should this be set trough the RPC interface when integrated?

Comment (guggero) [reply version.go:42]: We add `loopd` as a library in LiT and can simply set this global package-level variable. No need to add another RPC interface.

Comment (bhandras) [reply version.go:42]: I see, thanks for the explanation.

**Review by bhandras — APPROVED** [review]
LGTM :rocket:

File: version.go [hunk 2]
```diff
@@ -52,6 +62,25 @@ func Version() string {
...
+// that is submitting swaps to the loop server.
+func UserAgent() string {
+	// Start with the major, minor, and patch versions.
+	version := fmt.Sprintf("%d.%d.%d", appMajor, appMinor, appPatch)
+
+	// Append pre-release version if there is one.  The hyphen called for
+	// by the semantic versioning spec is automatically appended and should
+	// not be contained in the pre-release string.  The pre-release version
+	// is not appended if it contains invalid characters.
+	preRelease := normalizeVerString(appPreRelease)
```
Comment (carlaKC) [inline version.go:75]: nit: put this in a versionStr() function rather than duplicate what we already have in `Version`?

**Review by carlaKC — APPROVED** [review]
LGTM! 

Only one nit + could possibly squash the rpc changes with the commit where we actually set user agent string (since it's only 2 lines)

**Comment by carlaKC** [comment]
Hm just thinking, maybe also add this to `release_notes`? Makes it a little more likely that other devs building on loop will set the value?

**Review by alexbosworth — APPROVED** [review]

**Pull request merged**